adobe/aio-theme
# build: update theme to use yarn version 3.2.1

## Description

This PR updates the theme to use yarn package manager 3.2.1.

## Motivation and Context

Yarn 3 is the latest iteration of yarn and provides several bug fixes and benefits. You can find more information here: https://yarnpkg.com/getting-started/migration#why-should-you-migrate.

## How Has This Been Tested?

Tested on the `example` project within the theme as follows:

1. Run `yarn clean`.
2. Delete all `node_modules` directories.
3. Delete the `yarn.lock` file.
4. Run `yarn install`.
5. Run `yarn dev`.
6. Navigate through the `example` site.
7. Run `yarn clean` again.
8. Run `yarn build` to ensure a successful build.
9. Run `yarn serve` and navigate through the example site again.

Ran e2e tests (`yarn test:e2e`) on both main and yarn-update branches and got the same failures on my local environment. So I conclude that the updates to yarn 3 did not affect the e2e tests. :)

![image](https://user-images.githubusercontent.com/1828494/174818801-683da4ba-1035-4231-ad19-d488f1c79d3c.png)
![image](https://user-images.githubusercontent.com/1828494/174818864-64d04bdb-2d6a-4405-8aaf-ed86a3206861.png)


## Types of changes



- [x] Infrastructure change to build

## Checklist:
